roles/sap_ha_install_anydb_ibmdb2/tasks/main.yml

```diff
@@ -0,0 +1,165 @@
+    - "{{ sap_ha_install_anydb_ibmdb2_sid }}_HADR_2\t55002/tcp\t# DB2 HA Port 2" # DB2 HADR remote service (env var db2hadrremotesvc / HADR_REMOTE_SVC)
+  when: (sap_ha_install_anydb_ibmdb2_hostname_primary == inventory_hostname_short) or (sap_ha_install_anydb_ibmdb2_hostname_secondary == inventory_hostname_short)
+
+- name: SAP HA AnyDB - IBM Db2 HADR - Include tasks for Passwordless SSH
+  ansible.builtin.import_tasks: passwordless_ssh.yml
+
+
+- name: SAP HA AnyDB - IBM Db2 HADR - Ansible Task Block for IBM Db2 “Integrated Linux Pacemaker” configuration
+  when: (sap_ha_install_anydb_ibmdb2_hostname_primary == inventory_hostname_short) or (sap_ha_install_anydb_ibmdb2_hostname_secondary == inventory_hostname_short)
+  block:
+
+    # IBM Db2 LUW for SAP installation media includes already extracted Db2_*_Pacemaker_*_<<RHEL/SLES>><<pcmk_version>>_<<cpu_arch>>.tar.gz
+    # Default install file is /install_media/LINUXX86_64/ESE/disk1/db2/linuxamd64/pcmk/db2installPCMK
+    # Default RPM directory is /install_media/LINUXX86_64/ESE/disk1/db2/linuxamd64/pcmk/Linux
+    # Default SRPM (Source) directory is /install_media/LINUXX86_64/ESE/disk1/db2/linuxamd64/pcmk/Linux/SRPMS
+    - name: SAP HA AnyDB - IBM Db2 HADR - Identify IBM Db2 installation media with 'Integrated Linux Pacemaker' RPMs subdirectory
+      ansible.builtin.find:
+        paths: "{{ sap_ha_install_anydb_ibmdb2_software_directory }}"
+        recurse: true
+        file_type: file
+        patterns: db2installPCMK
+        excludes: bin
+      register: __sap_ha_install_anydb_ibmdb2_pcmk
+
+    - name: SAP HA AnyDB - IBM Db2 HADR - List all IBM Db2 'Integrated Linux Pacemaker' RPMs in subdirectory
+      ansible.builtin.find:
+        paths: "{{ (__sap_ha_install_anydb_ibmdb2_pcmk.files[0].path | dirname) + '/Linux/' + ('rhel' if ansible_os_family == 'RedHat' else 'suse' if ansible_os_family == 'Suse') }}"
+        recurse: true
+        file_type: file
+        patterns: "*.rpm"
+      register: __sap_ha_install_anydb_ibmdb2_pcmk_rpm_files
+
+    - name: SAP HA AnyDB - IBM Db2 HADR - Install all IBM Db2 'Integrated Linux Pacemaker' RPMs for RHEL
+      ansible.builtin.dnf:
+        name: "{{ __sap_ha_install_anydb_ibmdb2_pcmk_rpm_files.files | map(attribute='path') | list }}"
+        state: present
+        disable_gpg_check: true
+      when: ansible_os_family == 'RedHat'
+
+    - name: SAP HA AnyDB - IBM Db2 HADR - Install all IBM Db2 'Integrated Linux Pacemaker' RPMs for SLES
+      community.general.zypper:
+        name: "{{ __sap_ha_install_anydb_ibmdb2_pcmk_rpm_files.files | map(attribute='path') | list }}"
+        state: present
+        disable_gpg_check: true
+      when: ansible_os_family == 'Suse'
+
+    # SAP Note 3100287 - DB6: Db2 Support for Pacemaker Cluster Software
+    - name: SAP HA AnyDB - IBM Db2 HADR - Verify IBM Db2 'Integrated Linux Pacemaker' is installed
```

Comment:
What is the purpose of this and next task? Just to print package versions? They seem to be not influencing role login in any way.

Reply:
This comment suggests you did not run the Ansible Role, or the guidance that comes with it before you performed a (not requested) PR Review. It's advised in guidance to check the versions for IBM Db2 Support reasons.